utils/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/replicatedhq/embedded-cluster/utils
 
-go 1.24.4
+go 1.24.6
```

Comment:
I wonder if we should stop pinning the patch version in the dockerfiles? Let's discuss